apps/condo/domains/resident/schema/Resident.js

```diff
@@ -78,7 +78,16 @@ const Resident = new GQLListSchema('Resident', {
                     const [property] = await Property.getAll(context, { id: propertyId })
                     const residentAddress = getAddressUpToBuildingFrom(resolvedData.addressMeta)
                     if (property.address.toLowerCase() !== residentAddress.toLowerCase()) {
-                        return addFieldValidationError('Cannot connect property, because its address differs from address of resident')
+                        // but wait... if these addresses were updated by address service, the addressKey will be the same
+                        let areAddressesEqual = false
+
+                        if (!!property.addressKey && !!resolvedData.addressKey && property.addressKey === resolvedData.addressKey) {
+                            areAddressesEqual = true
+                        }
+
+                        if (!areAddressesEqual) {
```

Comment:
How it will affect `manageResidentToPropertyAndOrganizationConnections` which is invoked in `afterChange` hook of `Property`?

Reply:
The function works with `propertyId`. So, there will be no effect.